architecture-decision-records/0002-versioning-and-deployments.md

```diff
@@ -0,0 +1,75 @@
+# 2. Versioning and deployments
+
+**Date:** 2020-02-21
+
+## Context
+
+TDR will have multiple deployment environments: integration, staging and
+production. We want to be able to deploy new versions of our services to each
+environment in turn.
+
+## Decision
+
+Configure each project's build and Jenkinsfile so that when code is merged to
+master and the tests pass, Jenkins runs the following steps:
+
+1. Create a git tag with a version number based on the Jenkins build number, at
+   the commit that master is pointing to
+2. Push the version tag to GitHub
+3. If the repo is deployed as a Docker image (such as the [consignment
+   API][tdr-api] or the [frontend]), build the Docker image, tag it
+   with the same version number and push it to Docker Hub
+
```

Comment:
Might either need a comment to say we're moving to ECR or just change it to ECR because it will be right soon enough.

Reply:
I've added a comment saying we're moving to ECR.